modules/ximgproc/samples/disparity_filtering.cpp

```diff
@@ -0,0 +1,238 @@
+#include "opencv2/calib3d/calib3d.hpp"
```

Comment:
It should be `opencv2/calib3d.hpp`.

Header files `opencv2/<module>/<module>.hpp` are deprecated.